packages/clay-multi-step-nav/stories/index.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+import ClayMultiStepNav from '../src';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+const spritemap = require('@clayui/css/lib/images/icons/icons.svg');
+
+const ClayMultiStepNavWithState = () => {
+	const [value, setValue] = React.useState<number>(1);
+
+	const steps = [
+		{
+			active: value === 0,
+			complete: value > 0,
+			onClick: () => setValue(0),
+			subTitle: 'SubOne',
+			title: 'One',
+		},
+		{
+			active: value === 1,
+			complete: value > 1,
+			onClick: () => setValue(1),
+			subTitle: 'SubTwo',
+			title: 'Two',
+		},
+		{
+			active: value === 2,
+			complete: value > 2,
+			onClick: () => setValue(2),
+			subTitle: 'SubThree',
+			title: 'Three',
+		},
+		{
+			active: value === 3,
```

Comment:
I have a little bit of concern about defining `active` and `complete` directly on the items, I think it would be best to just set it directly in the `map` interaction.

Reply:
Similar to my comment above, I think it makes sense to do it hear at the low-level. Once its implemented into a high-level component, it'll likely be integrated.